x-pack/auditbeat/module/system/system.go

```diff
@@ -49,9 +52,16 @@ func NewModule(base mb.BaseModule) (mb.Module, error) {
 		return nil, err
 	}
 
+	log := logp.NewLogger(moduleName)
```

Comment:
The `BaseModule` should have its own logger now. So you don't have to create your own.

Reply:
Oh wait, it's the `BaseMetricSet` that got its own `Logger()`. Disregard.